src/test/java/de/agilecoders/wicket/demo/CustomRequireJsSettings.java

```diff
@@ -0,0 +1,94 @@
+package de.agilecoders.wicket.demo;
+
+import de.agilecoders.wicket.requirejs.AmdModulesRegistry;
+import de.agilecoders.wicket.requirejs.IRequireJsSettings;
+import de.agilecoders.wicket.webjars.request.resource.WebjarsJavaScriptResourceReference;
+
+import org.apache.wicket.ajax.json.JSONArray;
+import org.apache.wicket.ajax.json.JSONObject;
+import org.apache.wicket.request.Url;
+import org.apache.wicket.request.cycle.RequestCycle;
+import org.apache.wicket.request.resource.JavaScriptResourceReference;
+import org.apache.wicket.request.resource.ResourceReference;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Default implementation of {@link IRequireJsSettings}.
+ *
+ * @author miha
+ */
+public class CustomRequireJsSettings implements IRequireJsSettings {
+
+	private final AmdModulesRegistry registry;
+	private final String mountPath;
+	private final long waitSeconds;
+	private final ResourceReference resourceReference;
+
+	/**
+	 * Construct.
+	 */
+	public CustomRequireJsSettings() {
+		this.registry = new AmdModulesRegistry();
+		this.mountPath = "wicket/requirejs";
+		this.waitSeconds = TimeUnit.MINUTES.toSeconds(10);
+		this.resourceReference = new WebjarsJavaScriptResourceReference("requirejs/current/require.js");
+	}
+
+	@Override
+	public ResourceReference getResourceReference() {
+		return resourceReference;
+	}
+
+	@Override
+	public AmdModulesRegistry getModulesRegistry() {
+		return registry;
+	}
+
+	@Override
+	public String getMountPath() {
+		return mountPath;
+	}
+
+	@Override
+	public long getWaitSeconds() {
+		return waitSeconds;
+	}
+
+	@Override
+	public JSONObject getShimConfig() {
+		String KEY_DEPS = "deps";
+		String KEY_EXPORTS = "exports";
+
+		JSONObject deps = new JSONObject();
+		JSONObject shimGeneral = new JSONObject();
+		JSONArray GeneralDeps = new JSONArray();
```

Comment:
Please use camel cased names for variables.
In Java usually the variable names start with lower cased letter.